docs - update sql ref pages for GROUP/ROLE/USER commands

lisakowen opened the pull request:
this PR updates the listed sql command reference pages to align with the postgres v12 ref page content.  existing greenplum-specific content on the CREATE/ALTER role pages includes RESOURCE QUEUE and RESOURCE GROUP clauses and examples

questions:  
1.  do  the `ALTER ROLE <role_name> RESOURCE QUEUE {<queue_name> | NONE}`, and `ALTER ROLE <role_name> RESOURCE GROUP {<group_name> | NONE}` commands allow `<role_name>` to be `CURRENT_USER` or  `SESSION_USER`?  (i'll try to bring up a greenplum 7 cluster and test this out.)
2. postgres 12 ref pages did not include `DENY`-related clauses.  are these greenplum additions that i should add back in?

doc review site link for ALTER GROUP (behind vpn), can navigate to others from there:  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-gru/greenplum-database/GUID-ref_guide-sql_commands-ALTER_GROUP.html
doc review site link for ALTER ROLE (behind vpn), can navigate to others from there:  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-gru/greenplum-database/GUID-ref_guide-sql_commands-ALTER_ROLE.html
doc review site link for ALTER USER (behind vpn), can navigate to others from there:  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-gru/greenplum-database/GUID-ref_guide-sql_commands-ALTER_USER.html

huansong commented:
For the questions:

>do the ALTER ROLE <role_name> RESOURCE QUEUE {<queue_name> | NONE}, and ALTER ROLE <role_name> RESOURCE GROUP {<group_name> | NONE} commands allow <role_name> to be CURRENT_USER or SESSION_USER? (i'll try to bring up a greenplum 7 cluster and test this out.)

As you might've already tested it out - yes it can.

> postgres 12 ref pages did not include DENY-related clauses. are these greenplum additions that i should add back in?

Yes, they are Greenplum-only syntaxes and exist in 7X too.

lisakowen commented:
thank you for reviewing, @huansong .  i added the DENY clauses back in and adjusted the RESOURCE QUEUE/GROUP syntax.  i re-pushed the review site, you might need to hard refresh to see the changes.

huansong commented on gpdb-doc/markdown/ref_guide/sql_commands/ALTER_USER.html.md:
```diff
@@ -1,30 +1,17 @@
 # ALTER USER 
 
-Changes the definition of a database role \(user\).
+Changes the defintion of a database role.
```
typo: definition

huansong reviewed (approved):
LGTM!
